x-pack/plugins/security_solution/public/management/cypress/support/saml_authentication.ts

```diff
@@ -34,15 +35,15 @@ export const samlAuthentication = async (
       role: string | SecurityRoleName
     ): Promise<{ cookie: string; username: string; password: string }> => {
       // If config.env.PROXY_ORG is set, it means that proxy service is used to create projects. Define the proxy org filename to override the roles.
-      const rolesFilename = config.env.PROXY_ORG ? `${config.env.PROXY_ORG}.json` : undefined;
-      const sessionManager = new SamlSessionManager(
-        {
-          hostOptions,
-          log,
-          isCloud: config.env.CLOUD_SERVERLESS,
-        },
-        rolesFilename
-      );
+      const rolesFilename = config.env.PROXY_ORG
+        ? `${config.env.PROXY_ORG}.json`
+        : 'role_users.json';
+      const sessionManager = new SamlSessionManager({
+        hostOptions,
+        log,
+        isCloud: config.env.CLOUD_SERVERLESS,
+        cloudUsersFilePath: `${REPO_ROOT}/.ftr/${rolesFilename}`,
```

Comment:
For some reason not able to use `path.resolve` here: `Do not import Node.js builtin module "path"`

Let me know if there is the way to do it in a reliable way (this code will fail on Windows)

Reply:
Regarding change: I believe passing the absolute path to users file gives better flexibility and also less logic inside SamlSessionManager, which now will be used for both svl & stateful